bindings/python/python/opendal/exceptions.pyi

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class Error(Exception):
+    """OpenDAL unrelated errors"""
+
+    pass
+
+class UnexpectedError(Exception):
```

Comment:
Is it a good idea to remove the `Error` suffix of class name? So our users will use like:

```python
try:
    op.abc()
except opendal.exceptions.Unexpected as err:
    raise err
```

Reply:
SGTM